tests/acceptance/10_files/02_maintain/004.cf

```diff
@@ -49,6 +49,7 @@ body perms test_og
 
 body copy_from test_copy
 {
+      preserve => "true";
```

Comment:
OK, I reviewed this one. I think it's ok to change this test here to specify mode true. The policy was expecting the source and target modes to match, and it determined the expected mode by querying the source file for its mode.